infra/build/functions/target_experiment.py

```diff
@@ -279,15 +279,13 @@ def run_experiment(project_name, target_name, args, output_path, errlog_path,
   credentials, _ = google.auth.default()
   # Empirically, 3 hours is more than enough for 30-minute fuzzing cloud builds.
   build_lib.BUILD_TIMEOUT = 3 * 60 * 60
-  build_id = build_project.run_build(project_name,
-                                     steps,
-                                     credentials,
-                                     'experiment',
-                                     experiment=True,
-                                     extra_tags=[
-                                         f'experiment-{experiment_name}',
-                                         f'experiment-{project_name}'
```

Comment:
Removed the `experiment-` prefix to be consistent with tags from OFG.
Please let me know if keeping the prefix is preferred, I will fix OFG.